sae_lens/training/sparse_autoencoder.py

```diff
@@ -114,7 +124,13 @@ def forward(self, x: torch.Tensor, dead_neuron_mask: torch.Tensor | None = None)
             )
             + self.b_enc
         )
-        feature_acts = self.hook_hidden_post(torch.nn.functional.relu(hidden_pre))
+        if self.sae_type == "sae":
```

Comment:
I wonder if there's a cleaner way to do the forward pass that doesn't involve if statements (i expect this to slow the code down if we leave the if statements in)